Sources/Recycling/ListViews/FavoriteActions/Views/FavoriteActionHeaderView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+final class FavoriteActionHeaderView: UIView {
+    private lazy var imageView: UIImageView = {
+        let imageView = UIImageView(withAutoLayout: true)
+        imageView.layer.masksToBounds = true
+        imageView.contentMode = .scaleAspectFill
+        imageView.layer.cornerRadius = 12
+        return imageView
+    }()
+
+    private lazy var detailTextLabel: UILabel = {
+        let label = UILabel(withAutoLayout: true)
+        label.font = FavoriteActionHeaderView.detailTextLabelFont
+        label.textColor = .licorice
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.setContentHuggingPriority(.required, for: .vertical)
+        return label
+    }()
+
+    private lazy var accessoryLabel: UILabel = {
+        let label = UILabel(withAutoLayout: true)
+        label.font = FavoriteActionHeaderView.accessoryLabelFont
+        label.textColor = .licorice
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.setContentHuggingPriority(.required, for: .vertical)
+        return label
+    }()
+
+    private lazy var hairlineView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .sardine
+        return view
+    }()
+
+    // MARK: - Init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    // MARK: - Setup
+
+    func configure(withImage image: UIImage?, detailText: String, accessoryText: String) {
+        imageView.image = image
+        detailTextLabel.text = detailText
+        accessoryLabel.text = accessoryText
+    }
+
+    private func setup() {
+        backgroundColor = .milk
```

Comment:
Shouldn't the background be a light shade of gray?

Reply:
Apparently it should be a blur view 😄 I'll fix it.